content/news/playtest-20151031.md

```diff
@@ -0,0 +1,172 @@
+<p markdown="1">
+Halloween is upon us, and the latest OpenRA playtest is full of new tricks and other treats!
+This build includes all of the major new features and balance changes that we have planned for our Christmas release, and it is going to be one of our biggest releases yet!  Check out the
+[full changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/41b6f77d2b8c7307d5576e5521caeb25a0c2d476) for more details about the balance changes and new
+features.
```

Comment:
Why the new line? Also you already have "new features and balance changes" in the previous sentence. Can you find some different words?